source/Application/Controller/Admin/ShopLicense.php

```diff
@@ -117,8 +117,11 @@ protected function _fetchCurVersionInfo($sUrl)
         $oCurl->setMethod("POST");
         $oCurl->setUrl($sUrl . "/" . $sLang);
         $oCurl->setParameters($aParams);
-        $sOutput = $oCurl->execute();
-
+        try {
+            $sOutput = $oCurl->execute();
+        } catch (oxException $ex) {
```

Comment:
Isn't that a bit to general? Is there a more specific exception for this case?

Reply:
What is the error message of exception? maybe that message should be used? Maybe the resource is just not available, not because of firewall?